src/Application/GameConfig.cpp

```diff
@@ -11,14 +11,12 @@
 MM_DEFINE_ENUM_SERIALIZATION_FUNCTIONS(PlatformWindowMode, CASE_INSENSITIVE, {
     {WINDOW_MODE_WINDOWED, "windowed"},
     {WINDOW_MODE_BORDERLESS, "borderless"},
-    {WINDOW_MODE_FULLSCREEN, "fullscreen"},
     {WINDOW_MODE_FULLSCREEN_BORDERLESS, "fullscreen_borderless"},
     {WINDOW_MODE_FULLSCREEN_BORDERLESS, "borderless_fullscreen"}, // Alias for convenience
 
     // Compatibility with old serialization:
     {WINDOW_MODE_WINDOWED, "0"},
     {WINDOW_MODE_BORDERLESS, "1"},
-    {WINDOW_MODE_FULLSCREEN, "2"},
     {WINDOW_MODE_FULLSCREEN_BORDERLESS, "3"}
```

Comment:
Shouldn't this be set to 2 now? it was changed in src/Library/Platform/Interface/PlatformEnums.h but not here